src/main/java/org/folio/service/caches/MappingParametersCache.java

```diff
@@ -145,6 +206,115 @@ private List<Future<OrganizationCollection>> getOrganizationCollectionFutures(Ma
+  }
+
+  private Future<List<Location>> getLocations(OkapiConnectionParams params) {
+    String locationsUrl = "/locations?limit=" + settingsLimit;
+    return loadData(params, locationsUrl, LOCATIONS_RESPONSE_PARAM,
+      response -> response.mapTo(Locations.class).getLocations(), "Locations");
+  }
+
+  private Future<List<ContributorNameType>> getContributorNameTypes(OkapiConnectionParams params) {
+    String contributorNameTypesUrl = "/contributor-name-types?limit=" + settingsLimit;
+    return loadData(params, contributorNameTypesUrl, CONTRIBUTOR_NAME_TYPES_RESPONSE_PARAM,
+      response -> response.mapTo(Contributornametypes.class).getContributorNameTypes(), "ContributorNameTypes");
+  }
+
+  private Future<List<Fund>> getFunds(OkapiConnectionParams params) {
+    String fundsUrl = "/finance/funds?limit=" + settingsLimit;
+    return loadData(params, fundsUrl, FUNDS_RESPONSE_PARAM,
+      response -> response.mapTo(FundCollection.class).getFunds(), "Funds");
+  }
+
+  private Future<List<ExpenseClass>> getExpenseClasses(OkapiConnectionParams params) {
+    String fundsUrl = "/finance/expense-classes?limit=" + settingsLimit;
+    return loadData(params, fundsUrl, EXPENSE_CLASSES_RESPONSE_PARAM,
+      response -> response.mapTo(ExpenseClassCollection.class).getExpenseClasses(), "ExpenseClasses");
+  }
+
+  private Future<List<IdentifierType>> getIdentifierTypes(OkapiConnectionParams params) {
+    String identifierTypesUrl = "/identifier-types?limit=" + settingsLimit;
+    return loadData(params, identifierTypesUrl, IDENTIFIER_TYPES_RESPONSE_PARAM,
+      response -> response.mapTo(Identifiertypes.class).getIdentifierTypes(), "IdentifierTypes");
+  }
+
+  private Future<List<AcquisitionsUnit>> getAcquisitionsUnits(OkapiConnectionParams params) {
+    return acquisitionsUnitsService.getAcquisitionsUnits(null, 0, settingsLimit, new RequestContext(Vertx.currentContext(), params.getHeaders()))
+      .map(unitsCollection -> unitsCollection.getAcquisitionsUnits()
+        .stream().map(unit -> mapTo(unit, AcquisitionsUnit.class)).toList());
+  }
+
+  private Future<List<AcquisitionMethod>> getAcquisitionMethods(OkapiConnectionParams params) {
+    return acquisitionMethodsService.getAcquisitionMethods(settingsLimit, 0, null, new RequestContext(Vertx.currentContext(), params.getHeaders()))
+      .map(methodsCollection -> methodsCollection.getAcquisitionMethods()
+        .stream().map(method -> mapTo(method, AcquisitionMethod.class)).toList());
+  }
+
+  private static <T> T mapTo(Object unit, Class<T> objectClass) {
+    return Json.decodeValue(Json.encode(unit), objectClass);
+  }
+
+  private Future<List<String>> getTenantConfigurationAddresses(OkapiConnectionParams params) {
+    return RestUtil.doRequest(params, TENANT_CONFIGURATION_ADDRESSES_URL, HttpMethod.GET, null).compose(httpResponse -> {
+      if ((httpResponse.getResponse().statusCode() == HttpStatus.SC_OK)) {
```

Comment:
suggested to change to `if (httpResponse.getResponse().statusCode() == HttpStatus.SC_OK) {`

Reply:
Done, thank you